main/reference/ertp-api/purse.md

```diff
@@ -106,87 +90,87 @@ const depositAmountB = quatloosPurse.deposit(secondPayment, quatloos123);
+## Purse.getAllegedBrand()
+- Returns: **[Brand](./brand.md)**
 
+Returns an alleged brand (Note: a **Brand**, not a **String** as **allegedName()** methods do), 
+indicating what kind of digital asset the **Purse** purports to hold. This can identify the 
+**Purse**'s **Brand** if the **Purse** was made by a trusted **[Issuer](./issuer.md)** using **[Issuer.makeEmptyPurse()](./issuer.md#issuer-makeemptypurse)**.
 
-- `amount` `{Amount}`
-- Returns: `{Payment}`
+```js
+const PurseBrand = quatloosPurse.getAllegedBrand();
+```
 
-Withdraws the specified `amount` of digital assets from the `purse` into a new `payment` object.
+## Purse.getDepositFacet()
+- Returns: **DepositFacet**
 
-If the call succeeds, it immediately extracts the value into a new `payment`. 
-The caller won't get the new `payment` until a later turn, since the call is (nearly always) remote.
-But as soon as the message is processed, the value is gone from the `purse`.
+Creates and returns a **DepositFacet**, a new deposit-only facet of the **Purse** that allows 
+other parties to deposit **[Payments](./payment.md)** into the **Purse** without the ability
+to check the **Purse's** balance or withdraw from it.
+Theis makes it a safe way to let other people send you **Payments**.
 
+You can only deposit a **Payment** into a **DepositFacet** that's the same **[Brand](./brand.md)** as the original **Purse**.
+ 
 ```js
-// Create a purse and give it a balance of 10 Quatloos
-const { issuer, mint, brand } = makeIssuerKit('quatloos');
-const purse = issuer.makeEmptyPurse();
-const payment = mint.mintPayment(AmountMath.make(brand, 10n));
-const quatloos10 = AmountMath.make(brand, 10n);
-purse.deposit(payment, quatloos10);
+const depositOnlyFacet = Purse.getDepositFacet();
 
-// Withdraw an amount of 3 from the purse
-const quatloos3 = AmountMath.make(brand, 3n);
-const withdrawalPayment = purse.withdraw(quatloos3);
+// Give depositOnlyFacet to someone else. Anyone with a deposit facet reference can tell it to receive
+// a payment, thus depositing the payment assets in the Purse associated with the deposit facet.
+depositOnlyFacet.receive(payment);
+```
+Once you have created a **DepositFacet**, there is one method you can call 
+on it, **[DepositFacet.receive()](#depositfacet-receive-payment-optamount)**. The **DepositFacet** takes a **Payment** 
+and adds it to the balance of the **DepositFacet**'s associated **Purse**. The **Payment** 
+must be of the same **Brand** as what the **Purse** holds.
 
-// The balance of the withdrawal payment is 3 Quatloos
-issuer.getAmountOf(withdrawalPayment);
+Note the difference in method names for adding assets between a **Purse** and its **DepositFacet**.
+To add assets to a **Purse** directly, you use **Purse.deposit()**. To add assets
+to a **Purse** via its **DepositFacet**, you use **DepositFacet.receive()**.
 
-// The new balance of the purse is 7 Quatloos
-purse.getCurrentAmount();
-```
+## DepositFacet.receive(payment, optAmount?)
```

Comment:
I'm guessing that DepositFacet is implemented as an object with just the one method, and whatever Payment is successfully passed into it is automatically passed along to its "parent" Purse,, yes? Although that's how it may appear in the code, it's more helpful to present it to the reader as merely a facet of its parent Purse, even though that's not what it is, from a coding perspective. (What the hell is a "facet"??) Rather than listing it as a separate object, what I think it needs is a page that includes how to use it... something like an end-to-end scenario (with code), possibly something like "How to use a Purse". But that's a "nice to have" topic, rather than the meat & potatoes "must have" topics that I'm currently working on. So such a topic will have to be written at a later date.